### PR TITLE
add awesome-sec-talks that consists of various useful security talks

### PR DESCRIPTION
**Pull request by techgaun**
The repo for awesome-sec-talks is: https://github.com/PaulSec/awesome-sec-talks

### CODE REVIEW

**Comment by sindresorhus** [comment]
Can you fix the merge conflict?

The list is also missing a license, please read the [instructions](https://github.com/sindresorhus/awesome/blob/master/create-list.md) for creating a list and the [awesome manifest](https://github.com/sindresorhus/awesome/blob/master/awesome.md).

File: readme.md [hunk 1]
```diff
@@ -399,6 +399,7 @@ Check out my [blog](https://blog.sindresorhus.com) and follow me on [Twitter](ht
 - [For Girls](https://github.com/cristianoliveira/awesome4girls)
 - [Vorpal](https://github.com/vorpaljs/awesome-vorpal) *(Node.js CLI framework)*
 - [OKR Methodology](https://github.com/domenicosolazzo/awesome-okr) *(Goal setting & communication best practices)*
+- [Security Talks](https://github.com/PaulSec/awesome-sec-talks)
```
Comment (davisonio) [inline readme.md]: Could you also move the item under the 'Security' heading?

**Comment by techgaun** [comment]
Maybe @PaulSec can add the license. @davisonio I did update the section.

**Comment by PaulSec** [comment]
Hey guys, I might need to change some info from the repo since this is a *collected* list of security talks. We do not do any kind of selection or anything but try to reference as much videos as possible. 

Best,

**Comment by sindresorhus** [comment]
Alright. Closing for now. Ping me when it's ready and I'll reopen this PR.

**Pull request closed without merging**